Gemfile

```diff
@@ -1,5 +1,6 @@
 source "https://rubygems.org"
-
 gemspec
```

Comment:
L2 and L4 empty line was removed. I prefer those are reverted, and `Gemfile` is not modified in this PR.